### PR TITLE
Fixed 19449: Import History table alignment issue.

### PR DESCRIPTION
**Pull request by suryakant-krish**
### Description (*)

Import History table alignment issue.
### Fixed Issues (if relevant)

1. magento/magento2#19449: Import History table alignment issue.

### Manual testing scenarios (*)

1. Login into admin panel > System > import history > Import file "Download" alignment issue.

### Contribution checklist (*)
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds on Travis CI are green)

### CODE REVIEW

**Comment by magento-engcom-team** [comment]
Hi @suryakant-krish. Thank you for your contribution 
Here is some useful tips how you can test your changes using Magento test environment.
Add the comment under your pull request to deploy test or vanilla Magento instance:
- `@magento-engcom-team give me test instance` - deploy test instance based on PR changes
- `@magento-engcom-team give me 2.3-develop instance` - deploy vanilla Magento instance

For more details, please, review the [Magento Contributor Assistant](https://devdocs.magento.com/guides/v2.2/contributor-guide/contributing.html#contributor-assist) documentation

File: app/design/adminhtml/Magento/backend/Magento_Ui/web/css/source/module/_data-grid.less [hunk 1]
```diff
@@ -41,6 +41,11 @@
     overflow-x: auto;
     padding-bottom: 1rem;
     padding-top: @data-grid-header-row__indent;
+    .col-imported_file {
```
Comment (VladimirZaets) [inline app/design/adminhtml/Magento/backend/Magento_Ui/web/css/source/module/_data-grid.less]: Hi @suryakant-krish. Thanks for collaboration. Are you sure that hide <p> element is the best way to vertical align text?

Comment (suryakant-krish) [reply app/design/adminhtml/Magento/backend/Magento_Ui/web/css/source/module/_data-grid.less]: I have resolved this issue using php and removed my old css.

**Review by VladimirZaets — APPROVED** [review]

**Comment by magento-engcom-team** [comment]
Hi @VladimirZaets, thank you for the review.
[ENGCOM-3591](https://jira.corp.magento.com/browse/ENGCOM-3591) has been created to process this Pull Request

**Comment by VladimirZaets** [comment]
Hi @suryakant-krish. We made manual testing of the issue and we can't reproduce the original issue. Can you please provide more details? (Browser, etc)

Our steps to reproduce:
Go to System->Import
Import correct csv file
Go to System->Import History
 Imported File column contain 2 strings that are correct alignment

**Comment by suryakant-krish** [comment]
Hi @VladimirZaets. please follow steps without import any files.
Login into admin panel > system > Import History > 
Without any csv files uploaded download link misaligned. 
Please look into below attached screenshot.

**Comment by suryakant-krish** [comment]
Hello @VladimirZaets,
please review this issue.

**Comment by suryakant-krish** [comment]
Hello @VladimirZaets,
please review this issue.

**Comment by sdzhepa** [comment]
Hello @suryakant-krish 

Could you please clarify "preconditions"/"steps to reproduce" for this issue:

What exactly actions and steps required to receive this Actual Result?

In case without executed any Import before - we have empty grid, just with text : "We couldn't find any records."
emptyimportgrid


In case with some test import before(e.g. Products import) - Column "Imported File" has a name of used CSV file and link to Download

notemptyimportgrid

**Comment by VladimirZaets** [comment]
@suryakant-krish , I am closing this PR now due to inactivity.
Please reopen and update if you wish to continue.
Thank you for collaboration

**Pull request closed without merging**